### ISSUE TITLE
Stood on path triggers observer every tick

### ISSUE THREAD

**Issue by Fourmisain** [lebonq/Automatic-Path#32]
Turn on sound for this (and loop it)!

https://user-images.githubusercontent.com/8464472/128274418-13f222dc-788d-4e95-a9e6-fa60f36cbd31.mp4

I think this might be an easy fix.

Surely it's calling this each tick:

https://github.com/lebonq/Automatic-Path/blob/adf8b3a1e585e37652a0159d0de371103272a495/src/main/java/fr/lebon/autopath/mixin/BlockOnStepped.java#L22

By default `setBlockState()` uses the `Block.NOTIFY_ALL` flags, which is `NOTIFY_NEIGHBORS` and `NOTIFY_LISTENERS` combined.
I'm not sure exactly on the semantics of those flags, they are documented inside `Block`, it says:
```
/**
 * Sends a neighbor update event to surrounding blocks.
 */
public static final int NOTIFY_NEIGHBORS = 1;
/**
 * Notifies listeners and clients who need to react when the block changes.
 */
public static final int NOTIFY_LISTENERS = 2;
```
I'm assuming the neighbor updates are what does it, so this might be fixed by using
`world.setBlockState(pos,AutoPath.PATH_BLOCK.getDefaultState(), Block.NOTIFY_LISTENERS);`

**Comment by Fourmisain** [comment]
Ah, I got the wrong code snippet, the one above turns grass blocks to path, for which it is probably okay to notify neighbors - though I would change it too.

It's this one here which is the culprit:
https://github.com/lebonq/Automatic-Path/blob/cf9c30ecb200216562d25e7328196a7d57449551/src/main/java/fr/lebon/autopath/blocks/PathBlock.java#L57

**Comment by BrekiTomasson** [comment]
Any news on this issue? It's fairly annoying to have my sheep farms non-stop clicking like this. :)

**Comment by lebonq** [comment]
I think @Fourmisain got it right ! 
Right now I don't have time to dig more than this. So @Fourmisain if you can make a pull request  I will test it and release the jar ! 
Don't worry in the following week I'll fix more issue ;)

**Comment by lebonq** [comment]
Ok I have a strange issue with my hardware, Windows 11, Java 16 and Minecraft :(
I need to use my laptop so I'm a little bit late

**Comment by lebonq** [comment]
@BrekiTomasson Fix in 1.4.4 ! you can find it on github ! or curseforge in a couple of minutes
